packages/core/src/components/SearchBox/styles/customSearch.tsx

```diff
@@ -0,0 +1,14 @@
+import { styled } from '@medly-components/utils';
+
+export const CustomSearchFilterWrapper = styled.div<{ size: 'S' | 'M' }>`
+    box-shadow: 0 0.2rem 1.6rem 0 rgba(72, 79, 94, 0.2);
+    padding: 1.2rem;
+    border-radius: 0.8rem;
+    position: absolute;
+    left: 0;
+    top: ${({ theme, size }) => `calc(${theme.searchBox.height[size]} - 0.1rem) `};
+    width: 100%;
+    box-sizing: border-box;
+    z-index: 5;
```

Comment:
@gmukul01 what do you think of using higher z-index values? I've been preferring larger values, such as 50 or 100, so that we can easily position future elements in front of or behind existing elements without needing to make any changes.